docs/admin/access.rst

```diff
@@ -9,62 +9,52 @@ Access control
-for the whole instance, or in a limited scope.
+After installing Weblate, you can grant permissions to various things on the
+whole installation using :ref:`default-groups`. :ref:`acl` lets you grant
+permissions to specific translation projects. If you want to fine-tune the
+permissions you give based on groups you make yourself, you can do so
+using :ref:`custom-acl`.
 
-The permission system is based on groups and roles, where roles define a set of
-permissions, and groups assign them to users and translations, see
-:ref:`auth-model` for more details.
-
-After installation a default set of groups are created, and you can use those
-to assign users roles for the whole instance (see :ref:`default-groups`). Additionally when
-:ref:`acl` is turned on, you can assign users to specific translation projects.
-More fine-grained configuration can be achieved using :ref:`custom-acl`.
-
-Common setups
--------------
-
-Locking down Weblate
-++++++++++++++++++++
-
-To completely lock down your Weblate, you can use :setting:`REQUIRE_LOGIN` to
-force users to sign in and :setting:`REGISTRATION_OPEN` to prevent new
-registrations.
+In the permission system, roles define a set of permissions, and you can
+put users in groups that have one or more of these roles, described in
+:ref:`auth-model`.
 
 Site wide permissions
 +++++++++++++++++++++
 
-To manage permissions for a whole instance, just add users to `Users` (this is done
-by default using the :ref:`autogroup`), `Reviewers` and `Managers` groups. Keep
-all projects configured as `Public` (see :ref:`acl`).
+To manage permissions for a whole instance, just add users to 
+By default, new users get access to comment, provide suggestions, and make translations
+everywhere by being put in `Users` automatically by :ref:`autogroup`.
+You can put the users you deem fit in the `Reviewers` to let them review everything,
+and `Managers` groups to do all of the above everywhere.
+Keep all projects configured as `Public` (see :ref:`acl`).
 
 Per project permissions
 +++++++++++++++++++++++
 
-
 .. include:: /snippets/not-hosted-libre.rst
 
-Set your projects to `Protected` or `Private`, and manage users per
-project in the Weblate interface.
+Setting projects you administrate to `Protected` or `Private` lets you manage
+users per project in the Weblate interface.
 
 Custom permissions for languages, components or projects
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
 .. include:: /snippets/not-hosted-libre.rst
 
 Members are granted any permissions assigned to groups they are in, so you can
-grant the user multiple permissions at once. Create groups and attach them to a project,
+grant a user multiple permissions at once. Create groups and attach them to a project,
 component, or language. You can put users in multiple groups, and permissions
 can overlap between them.
 
-Granting any selected permissions based on project, component or language
```

Comment:
This whole paragraph was the initial problem. I messed it up, and it wasn't good before that. I can't understand how it adds anything of value, so removed it…